recipes/packio/all/conanfile.py

```diff
@@ -32,24 +32,31 @@ def _source_subfolder(self):
 
     @property
     def _compilers_minimum_version(self):
+        if scm.Version(self.version) < "2.4.0":
+            return {
+                "apple-clang": 10,
+                "clang": 6,
+                "gcc": 7,
+                "Visual Studio": 16,
+            }
         return {
-            "apple-clang": 10,
-            "clang": 6,
-            "gcc": 7,
+            "apple-clang": 13,
+            "clang": 11,
+            "gcc": 9,
             "Visual Studio": 16,
         }
 
     def config_options(self):
-        if tools.Version(self.version) < "1.2.0":
+        if scm.Version(self.version) < "1.2.0":
             del self.options.standalone_asio
-        if tools.Version(self.version) < "2.0.0":
+        if scm.Version(self.version) < "2.0.0":
             del self.options.msgpack
             del self.options.nlohmann_json
-        if tools.Version(self.version) < "2.1.0":
+        if scm.Version(self.version) < "2.1.0":
             del self.options.boost_json
 
     def requirements(self):
```

Comment:
Could you bump the dependencies to the newest version this library supports if it's not too much trouble please? :) 

The least I can do is to look up for you what the latest versions of each are:
`msgpack` is `3.3.0`. `nlohmann_json` is `3.11.2`, boost is `1.81.0` and `asio` is `1.27.0`